app/features/login/login-form.tsx

```diff
@@ -69,7 +69,12 @@ export function LoginForm({ onBack }: Props) {
               })}
             />
             {errors.email && (
-              <span role="alert" className="text-red-500 text-sm">
```

Comment:
I don't know why but when you apply `role="alert"` on span it doesn't use its content to build the accessible name (see [here](https://stackoverflow.com/questions/62763874/cannot-find-alert-by-accessible-name)) so I had to add  id and aria-labelledby that points to the id of the same element to get it to work.

I detected it because in tests I couldn't get the elements using getByRole selector